src/main/java/uk/ac/ebi/eva/pipeline/io/readers/NonAnnotatedVariantsMongoReader.java

```diff
@@ -36,7 +36,7 @@ public NonAnnotatedVariantsMongoReader(String databaseName, String collectionsVa
                                            MongoConnection connection) throws UnknownHostException {
         super();
 
-        setMongo(MongoDBHelper.getMongoClient(connection));
+        setMongo(new MongoDBHelper().getMongoClient(connection));
```

Comment:
Don't like the idea of creating a new instance of a class that we are injecting... The method is a helper one... but the class now is a configuration...

Maybe we should move the method to static?